### PR TITLE
React/react1/week1

### PR DESCRIPTION
**Pull request by AnastasiiaYaitska**

### CODE REVIEW

File: react/react1/week1/ToDoList/src/index.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+:root {
```
Comment (LucasIversen) [inline react/react1/week1/ToDoList/src/index.css:1]: good to see you play with styling in react also 👍

File: react/react1/week1/ToDoList/src/components/ToDoList/ToDoList.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ToDoItem from "../ToDoItem/ToDoItem";
+
+const ToDoList = ({ toDos }) => {
+  return (
+    <ol>
+      {toDos.map((todo) => (
```
Comment (LucasIversen) [inline react/react1/week1/ToDoList/src/components/ToDoList/ToDoList.jsx:7]: good use of the map operator

File: react/react1/week1/ToDoList/src/components/ToDoItem/ToDoItem.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./ToDoItem.css";
+
+const ToDoItem = ({ todo }) => {
```
Comment (LucasIversen) [inline react/react1/week1/ToDoList/src/components/ToDoItem/ToDoItem.jsx:4]: you're specifying which props are needed, which is a good way of making more robust components. good job

File: react/react1/week1/ToDoList/src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useState } from "react";
+import toDoList from "./assets/toDoList.json";
+import ToDoList from "./components/ToDoList/ToDoList";
+import "./App.css";
+
+function App() {
+  const [toDos, setToDos] = useState(toDoList);
```
Comment (LucasIversen) [inline react/react1/week1/ToDoList/src/App.jsx:8]: Good to see you use the useState hook. The todo list might make more sense as a prop, but nevertheless it's good to se that you've understood how to use states

**Review by LucasIversen — COMMENTED** [review]
Hi Anastasiia. Really good job! Keep it up 💪

File: react/react1/week1/ToDoList/src/App.jsx (hunk 4 above)
Comment (AnastasiiaYaitska) [reply react/react1/week1/ToDoList/src/App.jsx:8]: Thank you! 😊

Comment (AnastasiiaYaitska) [reply react/react1/week1/ToDoList/src/App.jsx:8]: Can I merge it?

Comment (LucasIversen) [reply react/react1/week1/ToDoList/src/App.jsx:8]: sure, go for it!

**Pull request merged**